### PR TITLE
mhf/ui-tweaks [DB-340]

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -47,23 +47,17 @@ const routes: Routes = [
           }
         },
         {
-          path: 'ehr/:id',
+          path: ':id',
```
Comment (mhigbyflowers) [inline public-ui/src/app/app-routing.module.ts:50]: /ehr/conditions -> /conditions

File: public-ui/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -47,23 +47,17 @@ const routes: Routes = [
...
           }
         }
         ]
       },
-
-      // {
-      //   path: 'quick-search/:dataType',
-      //   component: QuickSearchComponent,
-      //   data: { title: 'Quick Search' }
-      // },
```
Comment (mhigbyflowers) [inline public-ui/src/app/app-routing.module.ts:66]: no longer need this route

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 3]
```diff
@@ -420,7 +420,7 @@ export class ChartComponent implements OnChanges {
           ? 'pie' : 'bar',
         backgroundColor: 'transparent',
         style: {
-          fontFamily: 'Gotham'
+          fontFamily: 'Gotham-Book'
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/chart/chart.component.ts:423]: chart font works in all browsers

File: public-ui/src/app/styles/inputs.css [hunk 4]
```diff
@@ -52,23 +51,23 @@ input[type=url]:not([readonly]) {
...
 }
 .search-input-full input:-ms-input-placeholder { /* IE 10+ */
-  font-family: Gotham,open-sans,sans-serif;
+  font-family: 'Gotham A', 'Gotham B', 'Helvetica Neue', sans-serif;
   font-size: 16px;
   font-weight:bolder;
 }
 .search-input-full input:-moz-placeholder { /* Firefox 18- */
-  font-family: Gotham,open-sans,sans-serif;
+  font-family: 'Gotham A', 'Gotham B', 'Helvetica Neue', sans-serif;
```
Comment (mhigbyflowers) [inline public-ui/src/app/styles/inputs.css:70]: cross browser font

File: public-ui/src/app/styles/page.css [hunk 5]
```diff
@@ -120,7 +120,7 @@
 }
 
 .info-text {
-  font-size: .75em;
+  font-size: 14px;
```
Comment (mhigbyflowers) [inline public-ui/src/app/styles/page.css:123]: smaller font in domain tbl rows

File: public-ui/src/app/styles/inputs.css [hunk 6]
```diff
@@ -22,7 +22,6 @@ input[type=url]:not([readonly]) {
 .search-input-full {
   border-radius: 5px;
   background-color: #DAE6ED;
-  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.15);
```
Comment (mhigbyflowers) [inline public-ui/src/app/styles/inputs.css:25]: remove shadow to match mockup

File: public-ui/src/app/utils/db-config.service.ts [hunk 7]
```diff
@@ -80,14 +80,14 @@ export class DbConfigService {
...
+    'color': '#262262', 'font-family': 'Gotham-Book', 'font-size': '22px', 'font-weight': 'normal'
   };
   DATA_LABEL_STYLE = {
-    'color': '#f6f6f8', 'font-family': 'Gotham', 'fontSize': '15px', 'padding': '10px',
+    'color': '#f6f6f8', 'font-family': 'Gotham-Book', 'fontSize': '15px', 'padding': '10px',
     'font-weight': '300', 'textOutline': 'none',
   };
   GI_DATA_LABEL_STYLE = {
-    'color': '#f6f6f8', 'font-family': 'Gotham', 'font-size': '22px',
+    'color': '#f6f6f8', 'font-family': 'Gotham-Book', 'font-size': '22px',
```
Comment (mhigbyflowers) [inline public-ui/src/app/utils/db-config.service.ts:90]: cross-browser font

File: public-ui/src/app/views/ehr-view/ehr-view.component.css [hunk 8]
```diff
@@ -51,7 +50,7 @@
 
 .aka {
     padding-left:  9px ;
-    padding-top:18px;
+    padding-top:9px;
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.css:53]: smaller rows on domain tbl

File: public-ui/src/app/views/quick-search/quick-search.component.css [hunk 9]
```diff
@@ -1,5 +1,5 @@
 strong {
-    font-family: GothamMedium;
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/quick-search/quick-search.component.css:2]: cross-browser font

File: public-ui/src/app/views/quick-search/quick-search.component.ts [hunk 10]
```diff
@@ -66,7 +66,7 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
       'lifestyle' : this.tooltipText.lifestyleSurvey};
     pmConceptGroups: ConceptGroup[];
     conceptIdNames = [
-      { conceptId: 1585855, conceptName: 'Lifetyle' },
+      { conceptId: 1585855, conceptName: 'Lifestyle' },
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/quick-search/quick-search.component.ts:69]: fixed typo

**Review by mhigbyflowers — COMMENTED** [review]
Some UI tweaks

File: public-ui/src/app/views/breadcrumb/breadcrumb.component.css [hunk 11]
```diff
@@ -35,4 +35,5 @@
   color: #262262;
   font-weight: 500;
   text-decoration: none;
+  text-transform: capitalize;
 }
```
Comment (smazumd) [inline public-ui/src/app/views/breadcrumb/breadcrumb.component.css]: add one line

**Review by smazumd — APPROVED** [review]
Looks Good

**Pull request merged**